install/empty_data.php

```diff
@@ -9479,105 +9492,119 @@ public function getEmptyData(): array
         if ($add_e2e_data) {
             $root_entity = array_filter($tables['glpi_entities'], static fn($e) => $e['id'] === 0);
             $root_entity = current($root_entity);
+            $next_available_entity_id = 2;
```

Comment:
It should be computed with `max(array_column($tables['glpi_entities'], 'id'));` to be more resilient to changes.